### PR TITLE
GH34 | VBLOCKS-474

### PR DESCRIPTION
**Pull request by ryan-rowland**
@mhuynh5757 Addresses the bug filed here: #34

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+2.1.1 (In Progress)
+===================
+
+Bug Fixes
+---------
+
+- Ignoring a call will now properly stop the ringing sound
+- NPM versioning has been fixed to specify >=12 rather than exactly 12
```
Comment (mhuynh5757) [inline CHANGELOG.md:8]: Is this change included in this PR?

File: lib/twilio/call.ts [hunk 2]
```diff
@@ -1550,6 +1560,11 @@ namespace Call {
      */
     isUnifiedPlanDefault: boolean;
 
+    /**
+     * A function to be called after {@link Call.ignore} is called.
+     */
+    onIgnore: () => any;
```
Comment (mhuynh5757) [inline lib/twilio/call.ts:1566]: The other type is `() => void` (which I believe is preferred over `any`). Should we update this one to match?

File: CHANGELOG.md (hunk 1 above)
Comment (ryan-rowland) [reply CHANGELOG.md:8]: This was added on master and the CHANGELOG wasn't updated, so just adding it here.

File: lib/twilio/call.ts [hunk 3]
```diff
@@ -689,6 +698,7 @@ class Call extends EventEmitter {
     this._status = Call.State.Closed;
     this._mediaHandler.ignore(this.parameters.CallSid);
     this._publisher.info('connection', 'ignored-by-local', null, this);
+    this._onIgnore();
```
[the remark this answers is not in the document]
Comment (ryan-rowland) [reply lib/twilio/call.ts:701]: We always pass it, but you're right. We should treat `Call` as an independent class as it is. Will update.

**Pull request merged**